client_test.go

```diff
@@ -227,6 +229,40 @@ func TestGetNonexistingDoc(t *testing.T) {
 	check(t, "couchdb.NotFound(err)", true, couchdb.NotFound(err))
 }
 
+func TestBulkGet(t *testing.T) {
+	c := newTestClient(t)
+	c.Handle("POST /db/_bulk_get", func(resp http.ResponseWriter, req *http.Request) {
+		reqData := couchdb.BulkGet{}
+		body, _ := ioutil.ReadAll(req.Body)
+		err := json.Unmarshal(body, &reqData)
+		check(t, "reqData.Docs[0].ID", "foo", reqData.Docs[0].ID)
+		check(t, "reqData.Docs[1].ID", "bar", reqData.Docs[1].ID)
+		check(t, "reqData.Docs[2].ID", "baz", reqData.Docs[2].ID)
+		check(t, "json.Unmarshal", nil, err)
+
+		io.WriteString(resp, `{"results":[
+			{"id":"foo","docs":[{"ok":{"_id":"foo","_rev":"4-753875d51501a6b1883a9d62b4d33f91","field":1}}]},
+			{"id":"bar","docs":[{"ok":{"_id":"bar","_rev":"2-9b71d36dfdd9b4815388eb91cc8fb61d","field":2}}]},
+			{"id":"baz","docs":[{"error":{"id":"baz","rev":"undefined","error":"not_found","reason":"missing"}}]}]}`)
+	})
+
+	docs, notFound, err := c.DB("db").BulkGet([]string{"foo", "bar", "baz"}, testDocument{}, nil)
+	check(t, "err", nil, err)
+	check(t, "notFound", []string{"baz"}, notFound)
+
+	for _, doc := range docs {
```

Comment:
Please, update the test to check specifically that the first document is `foo` and the second one is  `bar`.

Reply:
fixed af2f14d1ae6ba5346047b6ee414664009beab2a9